e2e/localbitsong/scripts/setup.sh

```
@@ -19,6 +25,29 @@
+
+configure_keyring(){
+## remove old localbitsong key path
+rm -rf $CONFIG_FOLDER/test-keys
+## create new key path 
+mkdir -rf $CONFIG_FOLDER/test-keys
+## create new keys 
+bitsongd config keyring-backend test
+
+yes | $BIND  --home $BITSONG_HOME keys add $VAL --output json > $CONFIG_FOLDER/test-keys/$VAL.json 2>&1 
+sleep 1
+yes | $BIND  --home $BITSONG_HOME keys add $DEl1 --output json > $CONFIG_FOLDER/test-keys/$DEl1.json 2>&1 
```

Shellcheck (0.10.0)

[warning] DEl1 is referenced but not assigned (did you mean 'DEL1'?).

(SC2154)